flocker/node/_deploy.py

```diff
@@ -590,25 +590,30 @@ def find_differences(current_node_state):
             volumes = find_volume_changes(hostname, current_cluster_state,
                                           desired_state)
 
-            if volumes.resizing:
-                phases.append(InParallel(changes=[
-                    ResizeVolume(volume=volume)
-                    for volume in volumes.resizing]))
+            # if volumes.resizing:
+            #     # Can Openstack Volumes be resized? And if so, we
+            #     # probably don't need to support that in this spike.
+            #     phases.append(InParallel(changes=[
+            #         ResizeVolume(volume=volume)
+            #         for volume in volumes.resizing]))
 
             # Do an initial push of all volumes that are going to move, so
             # that the final push which happens during handoff is a quick
             # incremental push. This should significantly reduces the
             # application downtime caused by the time it takes to copy
             # data.
-            if volumes.going:
-                phases.append(InParallel(changes=[
-                    PushVolume(volume=handoff.volume,
-                               hostname=handoff.hostname)
-                    for handoff in volumes.going]))
+            # This can probably be ommitted for Openstack blocks...
+            # if volumes.going:
+            #     phases.append(InParallel(changes=[
+            #         PushVolume(volume=handoff.volume,
+            #                    hostname=handoff.hostname)
+            #         for handoff in volumes.going]))
```

Comment:
This may be the place to support moving volumes between regions or between cloud providers.